libs/features/personalization/personalization.js

```diff
@@ -70,7 +71,11 @@ export const normalizePath = (p) => {
     try {
       const url = new URL(path);
       const firstFolder = url.pathname.split('/')[1];
-      if (config.locale.ietf === 'en-US' || url.hash === '#_dnt' || firstFolder in config.locales || path.includes('.json')) {
+      if (!localize
+        || config.locale.ietf === 'en-US'
+        || url.hash === '#_dnt'
```

Comment:
I _think_ there's a slim chance that the hash would contain multiple hashs (e.g a modal)
```suggestion
        || url.hash.includes('#_dnt')
```